changelogs/current.yaml

```diff
@@ -39,6 +39,12 @@ behavior_changes:
   change: |
     Added new tag extraction so that scoped rds stats have their :ref:'scope_route_config_name
     <envoy_v3_api_msg_config/route/v3/scoped_route>' and stat prefix extracted.
+- area: http
+  change: |
+    Determine the order of the access and filter loggers, currently via a runtime guard.
+    When set ``true``, the filter loggers run before the access loggers to ensure any
+    dynamic metadata changes from filters are read properly. This change can be disabled
+    by setting the runtime guard flag ``envoy.reloadable_features.filter_access_loggers_first`` to ``false``.
```

Comment:
```
The access log handlers that added by the filters will be evaluated before the access 
log handlers that configured in the :ref:`access_log configuration
<envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.access_log>`.
This change can be disabled by setting the runtime guard flag 
``envoy.reloadable_features.filter_access_loggers_first`` to ``false``.
```

Reply:
Done. Added the test as well for review.